packages/appsync-modelgen-plugin/src/visitors/appsync-dart-visitor.ts

```diff
@@ -1197,26 +1174,25 @@ export class AppSyncModelDartVisitor<
    * @param dartCode
    */
   protected formatDartCode(dartCode: string): string {
-    if (this.isNullSafety()) {
-      return dartCode;
-    }
-    const result = dartStyle.formatCode(dartCode);
-    if (result.error) {
-      throw new Error(result.error);
-    }
-    return result.code || '';
+    return dartCode;
```

Comment:
The format function can be removed if it is not used any more (and its dependency)